sdk/digitaltwins/azure-digitaltwins-core/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 1.1.0-beta.1 (Unreleased)
```

Comment:
I don't know if you plan to do this as a separate PR, but the version needs to be changed to 1.0.1 too

Reply:
yes, I will send out a separate PR for release preparation

Reply:
on a second thought, I will include the release-preparation changes in the same PR